tools/github-actions/new-version/packaged-action/index.js

```diff
@@ -34055,7 +34055,7 @@ exports.formatGitTagsOutput = formatGitTagsOutput;
  * @param gitOutput response from git command to format
  */
 function formatGitTagsOutput(gitOutput) {
-    return gitOutput.split(/\s+/g)
+    return gitOutput.split(/[\r\n\s]+/g)
```

Comment:
`\s` does not already cover `\n` and `\r`?

Reply:
issue coming from the build of tools 
it will be fixed after the rebase 
I fallback all changes from tools